CODEOWNERS

```diff
@@ -9,3 +9,4 @@
 # Korean
 ko/ @revi
 pt-br/ @rffontenelle
+bg/ @j4ck-code
```

Comment:
The file itself looks okay, however I would add some formatting to it, so it just looks better

```suggestion
# Korean
ko/ @revi

# Brasilian Portugese
pt-br/ @rffontenelle

# Bulgarian
bg/ @j4ck-code
```